modules/gitops-release-notes-1-17-0.adoc

```diff
@@ -0,0 +1,154 @@
+
+[id="deprecation-of-some-fields-in-argo-cd_{context}"]
+=== Deprecation of Keycloak support in OpenShift GitOps
+
+* In {gitops-title} 1.17, the support for Keycloak-based authentication is deprecated. When Keycloak is configured for single sign-on (SSO), deprecation warnings appear in the logs. Dex is the default and recommended authentication provider. It integrates with the OpenShift OAuth server and supports authentication using OpenShift users and groups. Users are required to migrate to Dex-based authentication for continued support. To continue using Keycloak, you must deploy and manage their own link:https://docs.redhat.com/en/documentation/red_hat_build_of_keycloak/26.0/html/operator_guide/installation-[Red Hat Build of Keycloak (RHBK)] instance and manually configure the integration in the Argo CD custom resource. link:https://issues.redhat.com/browse/GITOPS-7070[GITOPS-7070]
+
+[id="removed-support-for-deprecated-initialRepositories-repositoryCredentials-fields_{context}"]
+=== Removed support for deprecated `initialRepositories` and `repositoryCredentials` fields
+
+* In {gitops-title} 1.17, support for the deprecated `initialRepositories` and `repositoryCredentials` fields has been removed. If these fields are used, Argo CD logs a warning indicating their removal. Users can add repositories and repository credentials manually using the Argo CD CLI or the Argo CD dashboard. link:https://issues.redhat.com/browse/GITOPS-5961[GITOPS-5961]
+
+[id="breaking-change-1-17-0_{context}"]
+== Breaking change
+
+[id="rbac-behavior-changes-with-argo-cd-dex-sso_{context}"]
+=== RBAC behavior changes with Argo CD 3.0 and Dex SSO
+
+* Argo CD 3.0 introduces a change in RBAC handling when using Dex for single sign-on (SSO). The RBAC system uses the `federated_claims.user_id` field instead of the previously encoded sub claim. This change affects how user identities are evaluated and matched in RBAC policies.
++
+To address this, the Argo CD Operator documentation has been updated with:
+
+** Detailed migration guidance for transitioning from sub to federated_claims.user_id
+** Step-by-step remediation procedures
+** Practical RBAC policy examples
+** Best practices for managing RBAC with the new hands-off approach
++
+Users must review and update their RBAC policies to ensure compatibility with Argo CD 3.0 when using the {gitops-title} Operator.
++
+--
+link:https://issues.redhat.com/browse/GITOPS-6887[GITOPS-6887]
+--
+
+[id="added-guidance-for-logs-rbac-enforcement-in-argocd_{context}"]
+=== Added guidance for logs RBAC enforcement in Argo CD 3.0
+
+* Before this update, the Argo CD Operator upstream documentation did not explain the changes to logs access control introduced in Argo CD 3.0. In contrast to Argo CD 2.x, where users with access to Applications could also view logs, Argo CD 3.0 treats logs as a distinct RBAC resource requiring explicit permissions. With this update, the Argo CD Operator upstream documentation includes:
```

Comment:
"...changes to logs access control...." to "...changes to logs RBAC..."  ? It's more specific.